.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           name: Database setup
           command: bin/rails db:schema:load --trace
       - browser-tools/install-browser-tools:
-          chrome-version: 116.0.5845.110 # https://github.com/CircleCI-Public/browser-tools-orb/issues/108#issuecomment-1958532112
+          chrome-version: 116.0.5845.187 # https://github.com/CircleCI-Public/browser-tools-orb/issues/108#issuecomment-1958532112
```

Comment:
To resolve Circle CI issue https://github.com/NCCE/teachcomputing.org-issues/issues/2795